src/main.rs

```diff
@@ -26,6 +28,7 @@ fn main() {
     opts.optopt("R", "", "changes the git remote used (default is origin)", "REMOTE");
     opts.optopt("b", "", "changes the base for merged branches (default is master)", "BRANCH");
     opts.optflag("h", "help", "print this help menu");
+    opts.optflag("", "version", "print the version");
```

Comment:
Good point, I hadn't thought about verbose.

At the moment there's not really any output that I'm silencing, so verbose doesn't seem like a thing that would get added anytime soon